helpers/tf_cfg.py

```diff
@@ -11,13 +11,31 @@
 __copyright__ = "Copyright (C) 2017-2019 Tempesta Technologies, Inc."
 __license__ = "GPL2"
 
+import logging
+
+from rich import pretty
+from rich.logging import RichHandler
+
+logger = logging.getLogger(__name__)
+
+log_levels = {
+    0: logging.CRITICAL,
+    1: logging.ERROR,
+    2: logging.WARNING,
+    3: logging.INFO,
+    4: logging.DEBUG,
+}
```

Comment:
```suggestion
log_levels = {0: 7, 1: 6, 2: 5, 3: 4, 4: 3, 5: 2, 6: 1}

for k, v in log_levels.items():
    logging.addLevelName(k, f"Lvl {v}")
```

See [comment](https://github.com/tempesta-tech/tempesta-test/pull/671#discussion_r1760689324)